crates/rpc/rpc-types/src/eth/engine/mod.rs

```diff
@@ -4,10 +4,13 @@
 
 mod cancun;
 mod forkchoice;
-mod payload;
+pub mod payload;
 mod transition;
-
 pub use self::{cancun::*, forkchoice::*, payload::*, transition::*};
+pub use payload::{
+    ExecutionPayload, ExecutionPayloadBodyV1, ExecutionPayloadFieldV2, ExecutionPayloadV1,
+    ExecutionPayloadV2, ExecutionPayloadV3, PayloadError, Withdrawal,
+};
```

Comment:
why wasn't the previous `pub use self::payload::*` sufficient here?

Reply:
right